app/javascript/lib/scp-api.jsx

```diff
@@ -11,16 +11,20 @@ import _compact from 'lodash/compact'
 import * as queryString from 'query-string'
 
 import { logJSFetchExceptionToSentry, logJSFetchErrorToSentry } from '~/lib/sentry-logging'
-import { getSCPContext } from '~/providers/SCPContextProvider'
 import { getAccessToken } from '~/providers/UserProvider'
 import {
   logDownloadAuthorization, logCreateUserAnnotation
 } from './scp-api-metrics'
 import { logSearch, mapFiltersForLogging } from './search-metrics'
 import { showMessage } from '~/lib/MessageModal'
+import {
+  clearOldServiceWorkerCaches, fetchServiceWorkerCache
+} from './service-worker-cache'
+import { getSCPContext } from '~/providers/SCPContextProvider'
+import { STEP_NOT_NEEDED } from './metrics-perf'
 
-
-const env = getSCPContext().environment
+// On each page load, check for old SCP caches, delete any found
+clearOldServiceWorkerCaches()
```

Comment:
I'd lean towards having this line of code in application.js.  I think it's a generally healthier pattern to have startup code there, and have modules not do anything automatically when they are loaded.  This saves us, for example, from running this method every time scp-api is imported in test